docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,234 @@
+2. Cluster Federation API server is running and network-reachable.
+
+See the [federation admin guide](/docs/admin/federation/) to learn
+how to bring up a cluster federation correctly (or have your cluster administrator do this for you), and how to correctly configure your client.
+
+#### I can create a Federated Service successfully against the Cluster Federation API, but no matching services are created in my underlying clusters
+Check that:
+
+1. Your clusters are correctly registered in the Cluster Federation API (`kubectl describe clusters`)
+2. Your clusters are all 'Active'.  This means that the cluster Federation system was able to connect and authenticate against the clusters' endpoints.  If not, consult the logs of the federation-controller-manager pod to ascertain what the failure might be. (`kubectl logs federation-controller-manager --namespace federation`)
+3. That the login credentials provided to the Cluster Federation API for the clusters have the correct authorization and quota to create services in the relevant namespace in the clusters.  Again you should see associated error messages providing more detail in the above log file if this is not the case.
+4. Whether any other error is preventing the service creation operation from succeeding (look for `service-controller` errors in the output of `kubectl logs federation-controller-manager --namespace federation`).
+
+#### I can create a Federated Service successfully, but no matching DNS records are created in my DNS provider.
+Check that:
+
+1. Your federation name, DNS provider, DNS domain name are configured correctly.  Consult the [federation admin guide](/docs/admin/federation/) or  [tutorial](https://github.com/kelseyhightower/kubernetes-cluster-federation) to learn
+how to configure your Cluster Federation system's DNS provider (or have your cluster administrator do this for you).
+2. Confirm that the Cluster Federation's service-controller is successfully connecting to and authenticating against your selected DNS provider (look for `service-controller` errors or successes in the output of `kubectl logs federation-controller-manager --namespace federation`)
+3. Confirm that the Cluster Federation's service-controller is successfully creating DNS records in your DNS provider (or outputting errors in it's logs explaining in more detail what's failing).
+
+#### Matching DNS records are created in my DNS provider, but clients are unable to resolve against those names
+Check that:
+
+1. The DNS registrar that manages your federation DNS domain has been correctly configured to point to your configured DNS provider's nameservers.  See for example [Google Domains Documentation](https://support.google.com/domains/answer/3290309?hl=en&ref_topic=3251230) and [Google Cloud DNS Documentation](https://cloud.google.com/dns/update-name-servers), or equivalent guidance from your domain registrar and DNS provider.
+
```

Comment:
Perhaps add "This troubleshooting guide did not help" and point them to the support channels, particularly mentioning slack?